ddp-workspace/projects/ddp-angio/src/styles.scss

```diff
@@ -1711,6 +1707,35 @@ select:-webkit-autofill {
     top: $warning-height !important;
 }
 
+// move user menu to the side panel on the small screens
+.user-menu-panel {
+    display: none;
+
+    .SimpleButton {
+        font-size: 1.1rem !important;
+        color: mat-color($app-theme, 600) !important;
+    }
+
+    .SimpleButton:hover {
+        color: mat-color($app-theme, 700) !important;
+    }
+}
+
+@media only screen and (max-width: 480px) {
+    .user-menu-panel {
+        display: block;
+
+        .ddp-user {
+            color: mat-color($app-theme, 600) !important;
+        }
+    }
+
+    .user-menu-header {
+        display: none;
+        padding: 0;
+    }
+}
```

Comment:
In smaller screens we are chopping off the hamburger.
<img width="368" alt="Screen Shot 2020-07-17 at 9 11 14 AM" src="https://user-images.githubusercontent.com/29984380/87789969-dad8e900-c80d-11ea-9e50-1d0e89e34177.png">

Reply:
Fixed 
![image](https://user-images.githubusercontent.com/30126907/87909539-1cb59980-ca71-11ea-8de2-fa3484ca6de3.png)